doc/source/whatsnew/v0.25.0.rst

```diff
@@ -28,6 +28,25 @@ Backwards incompatible API changes
 
 .. _whatsnew_0250.api.other:
 
+Indexing and getitem
+^^^^^^^^^^^^^^^^^^^^
```

Comment:
This is not exactly what I was after. Let's just do a single line for now.

Reply:
Sorry, I cannot understand what you mean.
Please tell me more concretely.